address/address.go

```diff
@@ -110,12 +109,6 @@ type Tap struct {
 	// asset to be made possible.
 	GroupKey *btcec.PublicKey
 
-	// groupWitness is a stack of witness elements that authorizes the
-	// membership of an asset in a partiular asset group. The witness can
-	// be a single signature or a script from the tapscript tree committed
-	// to with the TapscriptRoot, and follows the witness rules in BIP-341.
-	groupWitness wire.TxWitness
```

Comment:
Hmm, not sure why we ever had this....

Reply:
Me neither, but I'm guessing it was related to the check in asset commitments.